### PR TITLE
Adminpanel-v2 init

### PR DESCRIPTION
**Pull request by JohanAOstbye**
Created project skeleton with:
 - Dockerfile
 - Docker container
 - Basic routing
 - tailwind css
 - Component structure

also added adminpanel-v2 as a service in docker compose

### CODE REVIEW

File: packages/adminpanel-v2/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+> To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+
+
+## Contributing:
+
+### Branching
+Branching per issue
+
+Banchnavn mal: feature/adminpanel-**issuenr**
```
Comment (stormboat) [inline packages/adminpanel-v2/README.md:46]: Skal vi fortsatt kalle branchnavn 'feature/adminpanel-**issuenr**' eller 'feature/adminpanel-v2-**issuenr**' siden dere har kalt deres branch det?

**Review by stormboat — CHANGES_REQUESTED** [review]
Looks nice, only one question about the new branch naming convention.

**Review by yvindo — APPROVED** [review]

File: packages/adminpanel-v2/README.md (hunk 1 above)
Comment (JohanAOstbye) [reply packages/adminpanel-v2/README.md:46]: det henger bare igjen fra starten

Comment (stormboat) [reply packages/adminpanel-v2/README.md:46]: Konge, da ser alt bra ut!

**Review by stormboat — APPROVED** [review]
Looks good now

**Pull request merged**